doc/using/platform-support.chapter.md

```diff
@@ -1,18 +1,47 @@
 # Platform Support {#chap-platform-support}
 
-Packages receive varying degrees of support, both in terms of maintainer attention and available computation resources for continuous integration (CI).
+Packages receive varying degrees of support, both in terms of maintainer attention and available computation resources for continuous integration (CI). We have 7 defined tiers denoting how well supported each platform is.
 
-Below is the list of the best supported platforms:
+## Tiers {#sec-platform-tiers}
 
-- `x86_64-linux`: Highest level of support.
-- `aarch64-linux`: Well supported, with most packages building successfully in CI.
-- `aarch64-darwin`: Receives better support than `x86_64-darwin`.
-- `x86_64-darwin`: Receives some support.
+### Tier 1 {#sec-platform-tier1}
 
-There are many other platforms with varying levels of support.
-The provisional platform list in [Appendix A] of [RFC046], while not up to date, can be used as guidance.
+[Tier 1](https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#tier-1) platforms receive the highest level of support where problems can block updates, platform-specific patches are freely applied, and most packages are expected to work.
 
-A more formal definition of the platform support tiers is provided in [RFC046], but has not been fully implemented yet.
+### Tier 2 {#sec-platform-tier2}
 
-[RFC046]: https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md
-[Appendix A]: https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#appendix-a-non-normative-description-of-platforms-in-november-2019
+[Tier 2](https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#tier-2) platforms are expected to remain functional with updates, receive platform-specific patches as needed, and have many packages built by Hydra with full ofBorg support.
+
+### Tier 3 {#sec-platform-tier3}
+
+[Tier 3](https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#tier-3) platforms may receive non-intrusive platform-specific fixes, have native bootstrap tools available with cross-build toolchains in binary cache, but updates might break builds on these platforms.
+
+### Tier 4-7 {#sec-platform-tier4-7}
+
+Platform Tiers [4 through 7](https://github.com/NixOS/rfcs/blob/master/rfcs/0046-platform-support-tiers.md#tier-4) indicate varying levels of minimal support going from receiving only limited fixes to platforms with no support, but a path to support.
+
+## Breakdown {#sec-platform-breakdown}
+
+| Triple                        | Support Tier | Channel Blockers | User Adoption | Hydra Support | Ofborg Support | Bootstrap Tarballs | Cross Compiling Support |
```

Comment:
How can there be a boolean "user adoption"? What's that supposed to mean? Measured how, and by whom?

Reply:
Gotcha, should we drop the field or change it? If we were to have a user adoption field, it would convey that there's a good amount of people using the platform with nixpkgs. `aarch64-darwin` and `x86_64-linux-gnu` fits that bill. But in the future, it's possible `aarch64-linux` would be as well. Is there a way we could convey the same sort of thing but in a different way?

Reply:
I don't think a survey would be a good enough source of numbers for this sort of thing, because you'd get significant sampling bias with who responded considering how many users we have overall.  Even the framing would significantly impact the results, I'd expect. For example, musl users (of which we evidently have a fair few given the frequency of bug reports and fixes for static builds) likely wouldn't realise anything framed as a hardware survey was trying to count them.

Reply:
I don't see a good way of doing it at the moment, so I think we should omit it and reconsider it if in future we end up with some better data.